src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VersionFileCodeFlowUpdater.cs

```diff
@@ -605,20 +606,34 @@ await targetRepo.CommitAsync(
         ];
     }
 
-    private static List<DependencyUpdate> ComputeChanges(IReadOnlyCollection<string>? excludedAssets, VersionDetails before, VersionDetails after)
+    private static bool AssetExcluded(string name, IReadOnlyCollection<string>? excludedAssets)
     {
-        bool IsExcluded(DependencyDetail dep) => excludedAssets?.Contains(dep.Name) ?? false;
+        if (excludedAssets == null)
+        {
+            return false;
+        }
+        foreach (var excludedAsset in excludedAssets)
+        {
+            if (Regex.IsMatch(name, excludedAsset))
+            {
+                return true;
+            }
```

Comment:
Should we run a full regex? Maybe `.` in package names should be escaped? Maybe it does not matter too much but still

Reply:
If you just want to support wildcard then you wouldn't even need to depend on Regex here.

Reply:
Yeah, let's use https://learn.microsoft.com/en-us/dotnet/core/extensions/file-globbing